src/test/java/com/wikia/webdriver/testcases/globalnavigationbar/Navigating.java

```diff
@@ -10,18 +10,88 @@
 
-  private final String FANDOM_URL = "http://www.wikia.com/fandom";
-
-  @Execute(onWikia = "muppet")
   @Test(groups = {"fandomLogoClickOnEnCommunityOpensFandomWikia"})
   public void logoClickOnEnglishCommunityOpensFandom() {
     HomePage homePage = new HomePage();
     GlobalNavigation globalNav = homePage.getGlobalNavigation();
 
-    Assertion.assertTrue(globalNav.isFandomLogoVisible(), "Fandom logo not visible");
 
-    homePage = globalNav.clickWikiaLogo();
+    homePage = globalNav.clickFandomLogo();
+
+    Assertion.assertEquals(homePage.getCurrentUrl(), urlBuilder.getUrlForPage("fandom", "/"));
+  }
+
+  @Test(groups = {"gamesHubLinkClickOnEnCommunityOpensGamesHub"})
+  public void testGamesHubLink() {
+    HomePage homePage = new HomePage();
+    GlobalNavigation globalNav = homePage.getGlobalNavigation();
+
+    homePage = globalNav.clickGamesHubLink();
+
+    Assertion.assertEquals(homePage.getCurrentUrl(), urlBuilder.getUrlForPage("fandom", "/games"));
+  }
+
+  @Test(groups = {"moviesHubLinkClickOnEnCommunityOpensMoviesHub"})
+  public void testMoviesHubLink() {
+    HomePage homePage = new HomePage();
+    GlobalNavigation globalNav = homePage.getGlobalNavigation();
+
+    homePage = globalNav.clickMoviesHubLink();
+
+    Assertion.assertEquals(homePage.getCurrentUrl(), urlBuilder.getUrlForPage("fandom", "/movies"));
+  }
+
+  @Test(groups = {"tvHubLinkClickOnEnCommunityOpensTvHub"})
+  public void testTVHubLink() {
+    HomePage homePage = new HomePage();
+    GlobalNavigation globalNav = homePage.getGlobalNavigation();
+
+    homePage = globalNav.clickTVHubLink();
+
+    Assertion.assertEquals(homePage.getCurrentUrl(), urlBuilder.getUrlForPage("fandom", "/tv"));
+  }
+
+  @Test(groups = {"communityCentralLinkClickOnDeCommunityOpensDeCommunityCentral"})
+  @Execute(onWikia = "de.gta")
+  public void testCommunityCentralLinkOnDeCommunity() {
+    HomePage homePage = new HomePage();
+    GlobalNavigation globalNav = homePage.getGlobalNavigation();
+
+    homePage = globalNav.clickCommunityCentralLink();
+
+    Assertion.assertEquals(homePage.getCurrentUrl(), urlBuilder.getUrlForPage("de.community", "/wiki/Community_Deutschland"));
+  }
+
+  @Test(groups = {"exploreWikisLinkClickOnEnCommunityOpensExplorePage"})
+  public void testExploreWikisLink() {
+    HomePage homePage = new HomePage();
+    GlobalNavigation globalNav = homePage.getGlobalNavigation();
+
+    globalNav = globalNav.openWikisMenu();
+    homePage = globalNav.clickExploreWikisLink();
+
+    Assertion.assertEquals(homePage.getCurrentUrl(), urlBuilder.getUrlForPage("fandom", "/explore"));
+  }
+
+  @Test(groups = {"fandomUniversityLinkClickOnEnCommunityOpensFandomUniversity"})
+  public void testFandomUniversityLink() {
+    HomePage homePage = new HomePage();
+    GlobalNavigation globalNav = homePage.getGlobalNavigation();
+
+    globalNav = globalNav.openWikisMenu();
+    homePage = globalNav.clickFandomUniversityLink();
```

Comment:
You don't have to use homePage in `homePage.getCurrentUrl()` just use `driver.getCurrentUrl()`